app/views/layouts/application.slim

```diff
@@ -18,147 +18,146 @@ html
     = render partial: 'application/favicon'
     = render partial: 'shared/twitter_card'
     = content_for(:head)
-  body data-controller=controller_name data-action=action_name
     = Gon::Base.render_data
+  body data-controller=controller_name data-action=action_name
```

Comment:
Can you double check that this does not impact the contents of the global `gon` variable across page loads?  I remember something with turbolinks wanting this to not be in the head tag.

Reply:
Ah crap yeah you're right. It technically works but it's just appending more `window.gon=...` calls to `<head>`, growing the page source with every load. Moved back.